src/Shared/HashCodeCombiner/HashCodeCombiner.cs

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+#nullable enable
```

Comment:
Can we replace all usages of this class with https://docs.microsoft.com/en-us/dotnet/api/system.hashcode.combine?view=dotnet-plat-ext-3.1 instead?

Reply:
Once we move to net5. It doesn't exist in ns2.1